pufferpanel

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
-red=$(tput setf 4)
-green=$(tput setf 2)
-yellow=$(tput setf 6)
-blue=$(tput setf 1)
+red=$(tput setaf 1)
```

Comment:
As this is already included on master, please don't add this change as well. It will make the merge worse for us.